solutions/rcs-quickstart.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Step 1: Create 1 hub kind cluster and 2 managed Kind Clusters using a script from the OCM repository
+git clone https://github.com/open-cluster-management-io/OCM
+./OCM/solutions/setup-dev-environment/local-up.sh
+rm -rf OCM/
+
+# Step 2: Create the bin directory if it does not exist
+mkdir bin
```

Comment:
Check how we can avoid creating the bin directory here. If we cannot then clean it up